components/eam/src/physics/cam/modal_aer_opt.F90

```diff
@@ -1453,6 +1457,24 @@ subroutine modal_aero_sw(list_idx, dt, state, pbuf, nnite, idxnite, is_cmip6_vol
       call outfld('AODPROT',         protaod,    pcols, lchnk)
       call outfld('AODLIP',         lipaod,    pcols, lchnk)
 #endif
+
+#if defined(CLDERA_PROFILING)
+      call cldera_set_field_part_data("ABSORB" ,lchnk-begchunk+1,absorb)
+      call cldera_set_field_part_data("AODVIS" ,lchnk-begchunk+1,aodvis)
+      call cldera_set_field_part_data("AODALL" ,lchnk-begchunk+1,aodall)
+      call cldera_set_field_part_data("AODABS" ,lchnk-begchunk+1,aodabs)
+      call cldera_set_field_part_data("AODSO4" ,lchnk-begchunk+1,so4aod)
+      call cldera_set_field_part_data("BURDENSO4" ,lchnk-begchunk+1,burdenso4)
```

Comment:
Does the code have a problem with setting AODSO4 and BURDENSO4 given that they are now tagged quantities? I see that you defined them below this, but I just wasn't sure if lines 1466-1467 were necessary/problematic.

Reply:
Thanks for pointing this out because I think I was a bit confused when I was adding these. There are variables and outflds for AODSO4 and BURDENSO4:
https://github.com/sandialabs/CLDERA-E3SM/blob/f58d8e7b866513d64d4c8c9ebb381ef2bffbd4f1/components/eam/src/physics/cam/modal_aer_opt.F90#L1417
but I'm not sure what they actually compute. I was also able to track them with cldera-tools when I ran the test and they did have values.

Maybe it's best to leave them in there for now for possible backwards compatibility but not use them when running with tagging? These functions won't do anything unless we tell cldera-tools to track those quantities via an input file.

Reply:
That might be an error on my part when I was porting over the code. I'm thinking these will be one of the tagged species, but unsure which (i.e., first or last). Keeping them is fine as they aren't messing with the code function.